src/amo/components/Footer/index.js

```diff
@@ -8,21 +8,32 @@ import Link from 'amo/components/Link';
 import { makeQueryStringWithUTM, sanitizeHTML } from 'amo/utils';
 import translate from 'amo/i18n/translate';
 import Icon from 'amo/components/Icon';
+import type { I18nType } from 'amo/types/i18n';
 
 import './styles.scss';
 
-export class FooterBase extends React.Component {
-  static propTypes = {
-    _config: PropTypes.object,
-    i18n: PropTypes.object.isRequired,
-  };
+type Props = {|
+  noLangPicker?: boolean,
+|};
+
+type DefaultProps = {|
+  _config: typeof config,
+|};
+
+type InternalProps = {|
+  ...Props,
+  ...DefaultProps,
+  i18n: I18nType,
+|};
 
-  static defaultProps = {
+export class FooterBase extends React.Component<InternalProps> {
+  static defaultProps: {| ...Props, ...DefaultProps |} = {
     _config: config,
+    noLangPicker: false,
```

Comment:
Consider inverting that variable ? `showLangPicker` makes more sense to me, seems more in line with the rest of addons-frontend code, and it avoids the double negation here and in the `render()` below.

Reply:
The default is to have a langpicker so it reads better where I need the footer without the langpicker:

```
<Footer noLangPicker />
```

Instead of:

```
<Footer showLangPicker={false} />
```